datajunction-query/djqs/engine.py

```diff
@@ -128,6 +141,20 @@ def run_duckdb_query(
     return output
 
 
+def run_snowflake_query(
+    query: Query,
+    cur,
```

Comment:
minor: can we get the type of this cursor here?

Reply:
+1, `cur: snowflake.connector.cursor.SnowflakeCursor`

Reply:
@shangyian definitely, thanks for pointing that out.